CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing to rancher/windows
+
+Thank you for investing your time in contributing to our project!
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+## New contributor guide
+
+To get an overview of the project, read the [README](README.md). Here are some resources to help you get started with open source contributions:
+
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+
+## Getting started
+
+### Issues
+
+#### Create a new issue
+
+If you spot a problem with the docs, [search if an issue already exists](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests#search-by-the-title-body-or-comments). If a related issue doesn't exist, you can open a new [issue](https://github.com/rancher/windows/issues/new). 
+
+#### Solve an issue
+
+Scan through our [existing issues](https://github.com/rancher/windows/issues) to find one that interests you. If you find an issue to work on, you are welcome to open a PR with a fix.
```

Comment:
Would it make sense to have them indicate in some way that they intend on working on it? i.e. assign it to themselves or ask if it is available for outside contributions? Just so we aren't working on things in parallel without knowing

Reply:
While I agree with you on this, the actual file that it is this PR is just a straight copy paste from other repositories. 

I think this is worth discussing though as part of how we plan to work with community PRs as a Windows team (and in general in Rancher), but this is a larger discussion we can tackle after this PR is in.

Reply:
Oh okay, didn't realize it was a carbon copy. sounds good to me then